File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int Result[] = new int[2];
        HashMap<Integer, Integer> map = new HashMap<>();
         for(int i=0; i<nums.length; i++){
            int complement = target - nums[i];
            if(map.containsKey(complement)){
                Result[0] = map.get(complement);
                Result[1] = i;
                return Result;
            }
            else map.put(nums[i], i);
        }
        return Result;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        num_set = set(nums)
        longest = 0
        for n in nums:
            if n - 1 in num_set:
                continue
            i = 1
            while (n+i) in num_set:
                i += 1
            if longest < i:
                longest = i
        return longest
        